spec/functional/resource/archive_file_spec.rb

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright:: Copyright (c) Chef Software Inc.
```

Comment:
Since you wrote this all it's your copyright

Reply:
It's all good. Chef can have it 👍